README.tfdoc.hcl

```
@@ -138,18 +150,120 @@ section {
           configuration.
         END
       }
+    }
+
+    section {
+      title = "Extended Resource Configuration"
 
       variable "iam" {
-        type        = set(string)
+        type        = list(iam)
         default     = []
         description = <<-END
-          A list of iam members to gran publish access to the topic
+          List of IAM access roles to grant identities on the topic.
```

```suggestion
          List of IAM access roles to be granted to members on the topic.
```

From what I see we have role and then members that are assigned that role on the topic, so "grant identities" seemed confusing to me :thinking: (we are granting roles to members, not identities..I think, ignore if Im missing something stupid).